config/initializers/shopify_app.rb

```diff
@@ -1,13 +1,11 @@
 ShopifyApp.configure do |config|
+  config.application_name = "Triggerify"
   config.api_key = ENV['SHOPIFY_CLIENT_API_KEY']
   config.secret = ENV['SHOPIFY_CLIENT_API_SECRET']
+  config.old_secret = ""
   config.scope = "write_orders, write_products, write_customers, write_fulfillments, write_themes"
   config.embedded_app = true
-  config.webhooks = [
-    {
-      address: "#{Rails.configuration.application_url}/webhooks/app_uninstall",
-      format: 'json',
-      topic: 'app/uninstalled'
-    }
-  ]
```

Comment:
We'll need to check the impact here, as Shopify App now automatically resync webhooks by itself from time to time.